nservicebus/hosting/azure-functions/service-bus_queue-trigger-wiring_ASBFunctions_[1.3,).partial.md

```diff
@@ -0,0 +1,23 @@
+The Azure Function trigger for NServiceBus is auto-generated by specifying the logical endpoint name using a custom assembly attribute:
+
+```csharp
+[assembly: NServiceBusTriggerFunction("MyFunctionsEndpoint")]
+```
+
+The attribute will generate the trigger function required for NServiceBus:
+
+Note: An invalid endpoint name will generate an `NSBFUNC003` error with the message `Endpoint name is invalid and cannot be used to generate trigger function`.
```

Comment:
Do we want to elaborate on what is an invalid name?

Reply:
looking at the sources, this seems only to be invalid when it's null, empty or whitespace. Not sure this needs to be explicitly spelled out. Happy to further discuss but I'm not sure how to word that right now.